### PR TITLE
Create First missing positive

### PR DESCRIPTION
**Pull request by urvashi-2004**
Completed First missing positive problem

### CODE REVIEW

File: First missing positive [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		}
+   } 
+	return n + 1;
+}
+
+int main(){
+    int n=5;
+    int arr[] = { 1, 10, 2, 3, 4 };
+    cout << firstMissing(n,arr);
+}
```
Comment (md-shamim-ahmad) [inline First missing positive]: remove main function from this code.

File: First missing positive [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#include <bits/stdc++.h>
+using namespace std;
+
```
Comment (md-shamim-ahmad) [inline First missing positive]: remove the header and also namespace

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
Hey thanks for your PR.
But your code file path are not correct.
paste the code into the correct file path location.
For more help, [see the guide](https://github.com/md-shamim-ahmad/LeetCode-Problem-Solution/blob/main/Contribution%20Guidelines/How%20to%20Add%20a%20new%20Solution%20Code.md)

**Comment by urvashi-2004** [comment]
Hey @md-shamim-ahmad I have Changed the format. Please review my code and let me know if anything is wrong as I am contributing first time.

**Comment by urvashi-2004** [comment]
Hey @md-shamim-ahmad Let me know if anything is left. If not then please check and update to help me get my contribution counted. Thank you !!

File: First missing positive [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#41. [First Missing Positive](https://leetcode.com/problems/first-missing-positive/description/)
+
```
Comment (md-shamim-ahmad) [inline First missing positive]: replace this line
```#41. [First Missing Positive](https://leetcode.com/problems/first-missing-positive/description/)```
by 
```# [41. First Missing Positive](https://leetcode.com/problems/first-missing-positive/description/)```

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
hey @urvashi-2004
Your work are really awesome. But you have to change a little.

File name not correct.
If you want to add a leetcode question then file name will be ```README.md```.
As this file contains a leetcode code question then you rename this file as ```README.md``` .

And the path of this file is also incorrect.
Please see the [guidelines](https://github.com/md-shamim-ahmad/LeetCode-Problem-Solution/tree/main/Contribution%20Guidelines), then if you do not understand the comment again.

**Comment by urvashi-2004** [comment]
I have replace the line #41. [First Missing Positive](https://leetcode.com/problems/first-missing-positive/description/) by
# [41. First Missing Positive](https://leetcode.com/problems/first-missing-positive/description/)

And I rename my file as README.md. 

Can you tell me the correct file location so that I can resolve this issue because it is showing 404 page not found.

image

**Comment by md-shamim-ahmad** [comment]
change this path name or text ```First Missing Positive``` as

<br>
```LeetCode Problems/1 to 100/41. First Missing Positive/README.md```.

hey @urvashi-2004 , 
you just copy ```LeetCode Problems/1 to 100/41. First Missing Positive/README.md``` this text and paste it into ```First Missing Positive``` which I mark by  **read color** then, it will be good.


Please now check this,
[How to Add a new Solution Code](https://github.com/md-shamim-ahmad/LeetCode-Problem-Solution/blob/main/Contribution%20Guidelines/How%20to%20Add%20a%20new%20Solution%20Code.md#how-to-add-a-new-solution-code)
[How to Add a Problem README.md File](https://github.com/md-shamim-ahmad/LeetCode-Problem-Solution/blob/main/Contribution%20Guidelines/How%20to%20Add%20LeetCode%20Question.md#how-to-add-a-problem-readmemd-file)

**Comment by urvashi-2004** [comment]
Now, it's done. You can check and let me know if there is anything left.

**Comment by urvashi-2004** [comment]
Hey @md-shamim-ahmad Let me know if anything is left. If not then please check and update to help me get my contribution counted. Thank you !!

**Comment by md-shamim-ahmad** [comment]
hey @urvashi-2004 
Awesome job.
Do you want to add the solution code to the PR you created?

**Comment by urvashi-2004** [comment]
Yes @md-shamim-ahmad can you tell me how can I add solution

**Comment by md-shamim-ahmad** [comment]
check this link
[How to Add a new Solution Code](https://github.com/md-shamim-ahmad/LeetCode-Problem-Solution/blob/main/Contribution%20Guidelines/How%20to%20Add%20a%20new%20Solution%20Code.md#how-to-add-a-new-solution-code)

**Comment by urvashi-2004** [comment]
Thank you @md-shamim-ahmad for helping me. you really helped me a lot.

**Pull request merged**